docs/src/user-guide/guides-using-presto.md

```diff
@@ -31,8 +31,27 @@ Using Presto with CLP requires:
 
 ### Setting up CLP
 
-Follow the [quick-start](./quick-start/index.md) guide to set up CLP and compress your logs. A
-sample dataset that works well with Presto is [postgresql].
+1. Follow the [quick-start](./quick-start/index.md) guide to download and extract the CLP package.
+Before proceeding, navigate to `etc/clp-config.yml` in the package. Locate and uncomment the
+`database:` section, and change the `database.host` parameter to a non-localhost hostname/IP. The
+`database:` section should look like this:
```

Comment:
For readability in an editor, etc., we should indent these lines to the same level as the first line of the bullet.